js-peer/src/lib/libp2p.ts

```diff
@@ -114,6 +124,62 @@ export async function startLibp2p(): Promise<Libp2pType> {
   return libp2p
 }
 
+/* ability to Connect via peer-ID */
+export const findPeerById = (libp2p: Libp2pType) => async (peerIdStr: string) => {
```

Comment:
See https://github.com/libp2p/universal-connectivity/pull/210/files#r1952594020

This function is redundant. This logic is already implemented in js-libp2p.